traitsui/qt4/constants.py

```diff
@@ -42,17 +41,3 @@
 WindowColor = _palette.color(QtGui.QPalette.Window)
 
 del _palette
-
-# Screen size values:
-
-# QDesktopWidget.availableGeometry(int screen) is deprecated and Qt docs
-# suggest using screens() instead, but screens in not available in qt4
-if QtCore.__version_info__ >= (5, 0):
-    _geom = QtGui.QApplication.screens()[0].availableGeometry()
-else:
-    _geom = QtGui.QApplication.desktop().availableGeometry()
-
-screen_dx = _geom.width()
-screen_dy = _geom.height()
```

Comment:
With the module being publicly named, and the variable being equally public, I am not sure if there are no ones out there using these variables. @corranwebster are you sure we could remove them here?

Reply:
To err on the side of caution, shall we keep these constants here?

Reply:
I'm happy to leave them for now and just leave the issue open or open a new more specific issue to remove them from constants later